pyrasterframes/src/main/python/examples/NDVI.py

```diff
@@ -1,15 +1,19 @@
 from . import test_resource_dir
-from pyrasterframes.utils import create_rf_spark_session
-from pyrasterframes import *
-from pyrasterframes.rasterfunctions import *
+
+import pyrasterframes
+from pyrasterframes.rasterfunctions import rf_normalized_difference
+
 import os
 
-spark = create_rf_spark_session().withRasterFrames()
+spark = pyrasterframes.get_spark_session()
+
+red_path = os.path.join(test_resource_dir(), 'L8-B4-Elkton-VA.tiff')
+nir_path = os.path.join(test_resource_dir(), 'L8-B5-Elkton-VA.tiff')
```

Comment:
Would like to get rid of this `test_resource_dir` thing eventually.

Reply:
@vpipkt Head's up: in PR #143, the `examples` were moved into `docs`.

Reply:
great